### DISCUSSION TITLE
Unable to deploy with module path aliases on Vercel on next 9.4.4

### DISCUSSION THREAD

**Discussion by noobnoobdc137** [vercel/next.js#15492] [Help]
# Bug report

## Describe the bug

I use the feature [module aliases](https://nextjs.org/docs/advanced-features/module-path-aliases) but i cannot deploy  the project on Vercel

This is how my .jsconfig.json looks like
```json

{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "components": ["Components/*"],
      "globalStyles": ["public/static/styles/global/index.css"],
      "config": ["config"],
      "helpers": ["helpers"],
      "layouts": ["Layouts/*"]
    }
  }
}
```

My next.config.js
```js
// next.config.js
const withImages = require("next-images");
module.exports = withImages();
```

This is how i import in files
```jsx
import { RSWideCard, RSConnectBlock } from "components";
```

The configuration works perfectly on my development machine

- Windows 10
- WSL Ubuntu 18.04

but fails during deployment on Vercel

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Create a next js 9.4.4 project 
2. Create Module aliases using  .jsconfig.json 
3. export the project using :   "next build && next export",
4. Connect with Vercel and deploy

## Expected behavior

Project successfully deployed

## Screenshots

My Folder structure
![image](https://user-images.githubusercontent.com/55765209/88478752-741fa200-cf68-11ea-9c22-78cca6961564.png)

Err
![image](https://user-images.githubusercontent.com/55765209/88479132-a03c2280-cf6a-11ea-971e-afe225e344df.png)

## System information

- OS: Windows,WSL Ubuntu 18.04
- Version of Next.js: 9.4.4
- Version of Node.js: 14.5.0

## Additional context

Full Log

```
17:39:15 | Cloning github.com/noobnoobdc137/restart-website (Branch: master, Commit: 7876ea8)
-- | --
17:39:16 | Cloning completed in 782ms
17:39:16 | Analyzing source code...
17:39:18 | Installing build runtime...
17:39:19 | Build runtime installed: 386.908ms
17:39:19 | Looking up build cache...
17:39:19 | Build cache not found
17:39:20 | Installing dependencies...
17:39:20 | yarn install v1.22.4
17:39:20 | [1/4] Resolving packages...
17:39:20 | [2/4] Fetching packages...
17:39:30 | info fsevents@1.2.13: The platform "linux" is incompatible with this module.
17:39:30 | info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
17:39:30 | info fsevents@2.1.3: The platform "linux" is incompatible with this module.
17:39:30 | info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
17:39:30 | [3/4] Linking dependencies...
17:39:30 | warning "next-images > file-loader@6.0.0" has unmet peer dependency "webpack@^4.0.0 \|\| ^5.0.0".
17:39:30 | warning "next-images > url-loader@4.1.0" has unmet peer dependency "webpack@^4.0.0 \|\| ^5.0.0".
17:39:35 | [4/4] Building fresh packages...
17:39:35 | Done in 15.55s.
17:39:36 | Running "yarn export"
17:39:36 | yarn run v1.22.4
17:39:36 | $ next build && next export
17:39:36 | Creating an optimized production build...
17:39:36 | Attention: Next.js now collects completely anonymous telemetry regarding usage.
17:39:36 | This information is used to shape Next.js' roadmap and prioritize features.
17:39:36 | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
17:39:36 | https://nextjs.org/telemetry
17:39:42 | Failed to compile.
17:39:42 | ./pages/games.js
17:39:42 | Module not found: Can't resolve 'components' in '/vercel/6d136489/pages'
17:39:42 | > Build error occurred
17:39:42 | Error: > Build failed because of webpack errors
17:39:42 | at build (/vercel/6d136489/node_modules/next/dist/build/index.js:13:917)
17:39:42 | error Command failed with exit code 1.
17:39:42 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
17:39:42 | Error: Command "yarn export" exited with 1
17:39:44 | Done with "package.json"


```

**Comment by ACPixel** [comment]
Did you ever figure this out? I'm having the same issue.

**Reply by MichaelKim39** [reply]
Me too, would appreciate any advice

**Reply by forest1206** [reply]
Any updates on this issue?

**Reply by MichaelKim39** [reply]
Managed to solve this, actually it turned out the capitalisation was wrong on my import. Make sure that your import is 100% correct as many related issues seem to throw this error

**Reply by maxscn** [reply]
A year later, but in case it happens for anyone else, in my case it was Git which lowercased one of my file paths. I found this: https://vercel.com/support/articles/how-do-i-resolve-a-module-not-found-error which suggested to set it not to ignore casings with `git config core.ignorecase false`.

**Reply by design1online** [reply]
Gah, I spent 3 days trying to fix this!!! Ty @MaximilianSchon you are a lifesaver, your git change fixed it for me!

**Reply by rodoneill** [reply]
@MaximilianSchon saved me, thank you!

**Reply by heyFahad** [reply]
@MaximilianSchon, still helping folks even after 2.5 years. Kudos to your contribution mate 🙌🏼😄

**Reply by qjstuart** [reply]
@maxscn I was also saved by your comment yesterday. Thanks so much!

**Reply by reisdev** [reply]
Just tried this and it did the trick. Huge thanks!
